guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
+
+Discord.js now enforces and validates string input on all methods that expect it. Users will need to manually call `toString()` or utilise template literals for all string inputs as appropriate.
+
+The most common areas you will encounter this change is `MessageOptions#content` and the properties of a `MessageEmbed`, or when passing objects such as users or roles directly, especting them to be stringified.
+
+```diff
+- message.channel.send(user);
++ message.channel.send(user.toString());
+
+let count = 5;
+- embed.addField('Count', count);
++ embed.addField('Count', `${count}`);
+```
+
+### Intents
+
+As v13 makes the switch to Discord API v8, it will now be **required** to specify intents in your Client constructor. 
+
+They also move from `ClientOptions#ws#intents` to the top level `ClientOptions#intents`.
+
+Refer to our more [detailed article about this topic](/popular-topics/intents).
+
+```diff
+- const client = new Client({ ws: { intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });
++ const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });
+```
+
+### Structures#extend
+
+The concept of extendable Structures has been completely removed from Discord.js.
+
+For more information on why this decision was made and the recommended alternatives, refer to the [pull request](https://github.com/discordjs/discord.js/pull/6027).
+
+### Collectors
+
+All Collector related classes and methods (both create and await) now take a single object parameter which also includes the filter.
+
+```diff
+- const collector = message.createReactionCollector(filter, { time: 15000 });
++ const collector = message.createReactionCollector({ filter, time: 15000 });
+```
+
+### Naming conventions
+
+Some commonly used naming convention in Discord.js have been changed:
+
+#### Thing#thingId
+
+The casing of `thingID` properties has changed to `thingId`. This is a more-correct casing for the camelCase used by Discord.js as `Id` is an abbreviation, not an acronym.
+
+```diff
+- console.log(guild.ownerID);
++ console.log(guild.ownerId);
+
+- console.log(interaction.channelID);
++ console.log(interaction.channelId);
+```
+
+#### Client#message
+
+#### Client#interaction
+
+- `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+
+Don't worry - the old names still work, but you'll receive a Deprecation Warning until you switch over.
+
+```diff
+- client.on("message", message => { ... });
++ client.on("messageCreate", message => { ... });
+
+- client.on("interaction", interaction => { ... });
++ client.on("interactionCreate", interaction => { ... });
+```
+
+### Snowflakes
+
+For TypeScript users, discord.js now enforces the `Snowflake` type, a stringified bigint, rather than allowing any string to be accepted.
```

Comment:
```suggestion
For TypeScript users, Discord.js now enforces the `Snowflake` type, a stringified bigint, rather than allowing any string to be accepted.
```